public/docs/_examples/server-communication/dart/web/main.dart

```diff
@@ -0,0 +1,25 @@
+// #docregion
+import 'package:angular2/angular2.dart';
+import 'package:angular2/bootstrap.dart';
+import 'package:http/browser_client.dart';
+import 'package:http_in_memory_web_api/http_in_memory_web_api.dart';
+import 'package:server_communication/toh/toh_component.dart';
+import 'package:server_communication/hero_data.dart';
+import 'package:server_communication/wiki/wiki_component.dart';
+import 'package:server_communication/wiki/wiki_smart_component.dart';
+
+main() {
+  bootstrap(TohComponent, [
+//#enddocregion
+//#docregion in-mem-web-api-providers
+// in-memory web api providers
+    provide(BrowserClient,
+        useValue:
+            new HttpClientInMemoryBackendService(heroData)), // in-mem server
```

Comment:
This doesn't look ok to me. Where can I find the original source? Is it https://github.com/angular/angular.io/blob/77db0755c8ce8e001779937e5b890f13a3996484/public/docs/_examples/server-communication/ts/app/toh/toh.component.ts#L39?